recipes/zeus_expected/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.cmake import cmake_layout
```

Comment:
```suggestion
from conan.tools.cmake import cmake_layout
from conan.tools.microsoft import is_msvc
```

Reply:
Resolved by a [new upstream version](https://github.com/zeus-cpp/expected/releases/tag/v1.1.0) that don't need `"/Zc:__cplusplus` anymore.

Reply:
Thank you!!